### DISCUSSION TITLE
"New server version available" should only be shown to administrators

### DISCUSSION THREAD

**Discussion by Twisterado** [immich-app/immich#2870] [General]
The first time when opening the mobile app after a new server version has been released, every user gets a message, that a new server version is available and how to do the upgrade.
I think there should be a possibility to disable this message (maybe a checkbox "don't show again" or a server-side setting).
But more importantly non-admins should not get the message at all. They can't do anything about it and don't necessarily know what its trying to tell them.

**Comment by jrasm91** [comment]
I believe this has been implemented now.
